lib/matplotlib/_mathtext.py

```diff
@@ -1771,7 +1774,8 @@ class _MathStyle(enum.Enum):
 
     _dropsub_symbols = set(r'''\int \oint'''.split())
 
-    _fontnames = set("rm cal it tt sf bf default bb frak scr regular".split())
+    _fontnames = set("""rm cal it tt sf bf bfit
+                     default bb frak scr regular""".split())
```

Comment:
could be
```
_fontname = set("..."
                "...".split())
```
which seems nicer than a multiline string

Reply:
In fact, now that we have bumped the line length to 88, this all fits on one line within our guidelines, still